lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -90,37 +90,35 @@ def self.platform_from_app_id(app_id)
         end
       end
 
-      def self.binary_path_from_platform(platform, ipa_path, apk_path)
-        case platform
-        when :ios
-          ipa_path
-        when :android
-          apk_path
-        else
-          ipa_path || apk_path
+      def self.binary_path(platform, params)
+        if platform == :ios
+          return params[:ipa_path] ||
+                 Actions.lane_context[SharedValues::IPA_OUTPUT_PATH] ||
+                 Dir["*.ipa"].sort_by { |x| File.mtime(x) }.last
         end
-      end
 
-      def self.available_options
-        if lane_platform == :ios || lane_platform.nil?
-          ipa_path_default = Dir["*.ipa"].sort_by { |x| File.mtime(x) }.last
-        end
+        if platform == :android
```

Comment:
Maybe I don't understand ruby but wouldn't `params[:apk_path] || params[:android_artifact_path]` return a boolean?

Reply:
Good question. I feel like this is something I know but never actually think about, so I googled it and found a good explanation:

https://stackoverflow.com/a/1554379/530103

Reply:
TIL! Thanks :)